apps/backend/src/routes/login/+page.svelte

```diff
@@ -0,0 +1,161 @@
+<!-- JavaScript code -->
+<script lang="ts">
+  // import for forms
+  import { createForm } from 'svelte-forms-lib';
+
+  import { authStore } from '$lib/stores';
+  import type { FirebaseError } from 'firebase/app';
+
+  // import sign up function
+  import { signOut, signInEmailPassword } from '$lib/firebase/auth';
+
+  //import yup for form validation
+  import * as yup from 'yup';
+
+  let apiError: FirebaseError | null;
+
+  // Yup validation code
+  const { form, errors, handleChange, handleSubmit } = createForm({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validationSchema: yup.object().shape({
+      email: yup.string().email().required('Please enter your Email.'),
+      password: yup.string().required('Please enter your Password.'),
+    }),
+
+    onSubmit: async (data) => {
+      console.log(data);
+      try {
+        await signInEmailPassword(data.email, data.password, (error) => (apiError = error));
+      } catch (error) {
+        console.error(error);
+      }
+    },
+  });
+</script>
+
+<!-- Armadillo Logo-->
+<div class="flex items-center justify-center p-10">
+  <img src="armadillo.png" alt="" />
+</div>
+
+<div class="flex items-center justify-center">
+  <!-- Login Form - Card -->
+  <div class="w-full max-w-lg p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Login Form - Header -->
+    <p class="text-3xl font-semibold text-center mb-5">Welcome Back</p>
+
+    <!-- Login Form - Form Content -->
+    <form on:submit|preventDefault={handleSubmit}>
+      <!-- General Error Message -->
+      {#if apiError && (apiError.code === 'auth/user-not-found' || apiError.code === 'auth/wrong-password')}
+        <div class="p-4 flex flex-row bg-red-50 text-red-800 rounded-lg" role="alert">
+          <img src="errorlogo.svg" alt="" />
```

Comment:
A11y: Specify `alt` message instead of leaving it blank

Reply:
Put a generic error message here if thats okay: `alt="Error, Please Try Again Later."`